.github/workflows/Workflow.yml

```diff
@@ -39,3 +39,11 @@ jobs:
       APIM_DIGDIR_EMAIL: ${{ secrets.APIM_DIGDIR_EMAIL }}
     with:
       environment: test
+
+  deploy-slack-notifier-test:
+    needs: [deploy-test, check-for-changes]
+    uses: ./.github/workflows/DeployFunction.yml
+    if: ${{ needs.check-for-changes.outputs.hasSlackNotifierChanges == 'true' }}
```

Comment:
Konge! Etterhvert kan vi se på å få ut en matrix her istedenfor å lage en hasXChanged for hver tjeneste/tool (om det trengs da).